pulsar-testclient/src/main/java/org/apache/pulsar/testclient/PerformanceConsumer.java

```diff
@@ -200,6 +206,10 @@ public void received(Consumer consumer, Message msg) {
                     limiter.acquire();
                 }
 
+                long latencyMicros = MILLISECONDS.toMicros(System.currentTimeMillis() - msg.getPublishTime());
+                recorder.recordValue(latencyMicros);
```

Comment:
As mentioned in https://github.com/apache/incubator-pulsar/pull/898#discussion_r150306030, could you change these to following ?
```
long latencyMills = System.currentTimeMillis() - msg.getPublishTime();
recorder.recordValue(latencyMills);
```

Reply:
I thought that using is wrong, because recorder used value is in the microseconds level. When I record milliseconds directly, the output is less than 0.01, which is not believable. The recorder used in PerformanceProducer records micro seconds as well.

Reply:
Sorry for  my careless fault, I have fixed it.